pkgs/by-name/br/brave/update.sh

```diff
@@ -3,15 +3,13 @@
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
 
-latestVersionAarch64="$(curl -sL https://brave-browser-apt-release.s3.brave.com/dists/stable/main/binary-arm64/Packages \
-    | sed -r -n 's/^Version: (.*)/\1/p' | head -n1)"
+latestVersion="$(curl --fail -s ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} "https://api.github.com/repos/brave/brave-browser/releases/latest" | jq -r '.tag_name' | sed 's/^v//')"
```

Comment:
Do you need to add jq to the packages in nix-shell at the top of this file?

Reply:
Good catch. Updated.